app/templates/main/results.html

```diff
@@ -55,6 +55,35 @@
             {{ f.begin_form(cba, flashes) }}
             {{ f.render_form(cba) }}
             {{ f.end_form(cba) }}
+            <br />
+            <button class="ui button" 
+            href="/search" 
+            style="font-family: Avenir; width: 80%; margin-bottom: 20px; text-align: center"
+            onclick="window.resultList.sortByDateDesc()">
+               Date (Descending)
+            </button>
+            
+            <button class="ui button" 
+            href="/search" 
+            style="font-family: Avenir; width: 80%; margin-bottom: 20px; text-align: center"
+            onclick="window.resultList.sortByDateAsce()">
+               Date (Ascending)
+            </button>
+
+            <button class="ui button" 
+            href="/search" 
+            style="font-family: Avenir; width: 80%; margin-bottom: 20px; text-align: center"
+            onclick="window.resultList.sortByPaidDesc()">
```

Comment:
for this - I made the Result List (a React component) a global variable by attaching it to the Browser window. This means I can directly call on the functions within React component via a button.